runtime/themes/darcula.toml

```diff
@@ -28,6 +28,8 @@
 "ui.virtual.jump-label" = { fg = "lightblue", modifiers = ["italic", "bold"] }
 "ui.bufferline" = { fg = "grey04", bg = "grey00" }
 "ui.bufferline.active" = { fg = "grey07", bg = "grey02" }
+"ui.picker.header.column" = { fg = "grey05", modifiers = ["italic", "bold"] }
+"ui.picker.header.column.active" = { fg = "grey05", modifiers = ["italic", "bold"] }
```

Comment:
The `.active` one can be removed: when we look up `ui.picker.header.column.active` it will fall back to `ui.picker.header.column`

Reply:
Actually, I changed the active column to work as intended, to highlight the current search column.
![hx after2](https://github.com/user-attachments/assets/fc10084b-f4e4-4c55-bd23-c3bc425c6acf)